javascripts/discourse-linkify/lib/utilities.js

```diff
@@ -31,8 +32,17 @@ const escapeRegExp = function (str) {
 };
 
 const prepareRegex = function (input) {
-  let leftWordBoundary = "(\\s|[:.;,!?…\\([{]|^)";
-  let rightWordBoundary = "(?=[:.;,!?…\\]})]|\\s|$)";
+  let leftWordBoundary = "(\\b)";
```

Comment:
@md-misko Please keep the default as it was. `\b` in javascriopt is horribly implemented and does not work with non-ascii alphabets.